Add news page (#113)
genome-nexus/genome-nexus-frontend: src/doc/News.md

## Dec 16, 2021

-   **API Update**: New search endpoint to search Genome Nexus database.
    -   For example, search [BRAF p.V600E](https://www.genomenexus.org/search?keyword=BRAF%20p.V600E)

## Nov 04, 2021

> -   **Website New Feature**: Add UniProt topology track to variant page. \* [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)

Reviewer: Maybe just remove `\*` here and in the other places? It looks a bit like a footnote indicator
```suggestion
-   **Website New Feature**: Add UniProt topology track to variant page. [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
```

Author: @inodb This is added by prettier, so I add `md` file to `.prettierignore`

## May 21, 2020

-   **API Update**: Add [SIGNAL](https://www.signaldb.org/) annotation field for annotation endpoint.
    -   [Example: 7:g.55249071C>T](https://www.genomenexus.org/annotation/7%3Ag.55249071C%3ET?fields=signal)

## May 21, 2020

-   **Website New Feature**: Support transcript switch on variant page.
    -   [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
        <img src="https://user-images.githubusercontent.com/16869603/148014139-1974e152-edf1-4d95-9dd7-2ad52167e954.png" />

## Mar 25, 2020

-   **API Update**: Add nucleotide context endpoints.
    -   [Example: 7:g.140453136A>T](https://www.genomenexus.org/nucleotide_context/7%3Ag.140453136A%3ET)

## Mar 05, 2020

-   **API Update**: Add amino acids to annotation summary.
    -   [Example: 7:g.55249071C>T](https://www.genomenexus.org/annotation/7:g.55249071C%3ET?fields=annotation_summary)

## Mar 03, 2019

-   **Command Line Interface Update**: Add OncoKB annotation to output annotation MAF file.

## Feb 04, 2020

-   **API Update**: Add [OncoKB](https://www.oncokb.org/) to annotation endpoints.

>     -   Please get your license fron [OncoKB](https://www.oncokb.org/) first and paste your `token` to API.

Reviewer 2: ```suggestion
    -   Please get your license from [OncoKB](https://www.oncokb.org/) first and paste your `token` to API.
```

## Dec 03, 2019

-   **Website New Feature**: Add Functional prediction and Population prevalence to variant page. \* [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
    <img src="https://user-images.githubusercontent.com/16869603/148015515-2fe48e0b-da87-4406-a46e-d877bf524bcb.png" />

## Oct 18, 2019

-   **Command Line Interface Update**: Command line interface for Genome Nexus annotation and convertion.
    -   For more information, see [genome-nexus-cli](https://github.com/genome-nexus/genome-nexus-cli)

## Sep 18, 2019

-   **Website New Feature**: Add mutation mapper lollipop to variant page. \* [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
    <img src="https://user-images.githubusercontent.com/16869603/148015718-c0cfe2bb-dc6a-4a7d-b4b6-71a65e3e635a.png" />

## Sep 18, 2019

-   **Website New Feature**: Add variant page. \* [Example: 7:g.55249071C>T](https://www.genomenexus.org/variant/7:g.55249071C%3ET)
    <img src="https://user-images.githubusercontent.com/16869603/148015822-887a52f4-bfa3-4ae9-b00d-7e27aff90bd3.png" />

## Aug 15, 2019

> -   **API Update**: Add an API end point for MSK Insight (SIGNAL) mutations.

Reviewer 2: ```suggestion
-   **API Update**: Add an API endpoint for MSK Insight (SIGNAL) mutations.
```
